kotlin/src/test/kotlin/org/axonframework/extensions/kotlin/serialization/KotlinSerializerTest.kt

```diff
@@ -0,0 +1,91 @@
+package org.axonframework.extensions.kotlin.serialization
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
+import org.axonframework.serialization.AnnotationRevisionResolver
+import org.axonframework.serialization.ChainingConverter
+import org.axonframework.serialization.SerializedType
+import org.axonframework.serialization.SimpleSerializedObject
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class KotlinSerializerTest {
+
+    /**
+     * This class will automatically become serializable through the Kotlin serialization compiler plugin.
+     */
+    @Serializable
+    data class TestData(
+        val name: String,
+        val value: Float?
+    )
+
+    @Test
+    fun canSerializeTo() {
+        val serializer = kotlinSerializer()
+
+        assertTrue(serializer.canSerializeTo(String::class.java))
+        assertTrue(serializer.canSerializeTo(JsonElement::class.java))
+    }
+
+    @Test
+    fun `configuration options`() {
+        val serializer = kotlinSerializer {
+            json = Json
+            converter = ChainingConverter()
+            revisionResolver = AnnotationRevisionResolver()
+        }
+        assertNotNull(serializer)
+    }
+
+    @Test
+    fun serialize() {
```

Comment:
Since `serialize` can accept `null` value as an input, that case should be also verified

Reply:
Indeed! I forgot it for the serialization case :)